packages/cli/src/commands/source/push/add.ts

```diff
@@ -23,19 +21,22 @@ export default class SourcePushAdd extends Command {
 
   public static flags = {
     file: Flags.string({
+      // For retro compatibility
       multiple: true,
-      exclusive: ['folder'],
-      char: 'f',
-      helpValue: 'myfile.json',
-      description: 'One or multiple file to push. Can be repeated.',
+      hidden: true,
     }),
     folder: Flags.string({
+      // For retro compatibility
       multiple: true,
-      exclusive: ['file'],
       char: 'd',
-      helpValue: './my_folder_with_multiple_json_files',
+      hidden: true,
+    }),
+    files: Flags.string({
+      multiple: true,
+      char: 'f',
+      helpValue: 'myfile.json',
       description:
-        'One or multiple folder containing json files. Can be repeated',
+        'Combinaison of JSON files and folders (containing JSON files) to push. Can be repeated.',
     }),
```

Comment:
1. Ensure that `file` and `folder` are aliases to `files` 
2. Create an Epic 'CLI-v2.0' and a User Story for this specific deprecation (i.e. removing the file and folder flags
3. Add a //TODO with the Identifier of the User Story

Reply:
> Instead of specifying the source as a args, and the files as flags, we could do the opposite.

You don't need to specify the flag multiple time. It is
`coveo source:push:new mySource -f myFirstFile mySecondFile myThirdFile`

1. Can we create aliases for flags with oclif?

Reply:
No, but what you do is that you ensure all flags are mutually exclusive, and then you treat 'em all the same (i.e. dir now support file and file support dir, but that's fine it's not breaking :P )

Reply:
At the end, it really doesn't matter what flag you use because the SDK checks whether the entry is a file or directory.
So in CDX-856, we can remove all flags.